File: 프로그래머스/lv1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.py
def solution(arr):
    answer = [0] * len(arr)
    for i in range(len(arr)):
        answer[i] = arr[i]
    arr.sort(reverse=True)
    if len(arr) == 1:
        answer = [-1]
    else:
        idx = answer.index(arr[-1])
        answer.pop(idx)
    return answer

File: 프로그래머스/lv1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.cpp
#include <string>
#include <vector>
#include <iostream>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> arr) {
    vector<int> answer;
    int minV = arr[0];
    if (arr.size() == 1){
        answer.push_back(-1);
    }
    else {
     for (int i; i < arr.size(); i++){
         if (arr[i] < minV){
             minV = arr[i];
         }
     }
        
    for (int i; i< arr.size(); i++){
        if (arr[i] != minV){
            answer.push_back(arr[i]);
        }
    }
    }
    return answer;
}